pkg/cmd/roachtest/github_test.go

```diff
@@ -103,6 +105,27 @@ func TestShouldPost(t *testing.T) {
 	}
 }
 
+func TestGenerateHelpCommand(t *testing.T) {
+	vo := vm.DefaultCreateOpts()
+	vmOpts := &vo
+
+	reg := makeTestRegistry(spec.GCE, "", "", false, false)
+	clusterSpec := reg.MakeClusterSpec(1, spec.Arch("arm64"))
+	testClusterImpl := &clusterImpl{name: "foo-cluster", spec: clusterSpec, arch: vm.ArchAMD64}
+
+	github := &githubIssues{
+		vmCreateOpts: vmOpts,
+		cluster:      testClusterImpl,
+		teamLoader:   validTeamsFn,
+	}
+
+	r := &issues.Renderer{}
+	generateHelpCommand(github)(r)
+
+	// TODO (BEFORE MERGE): time.Now() in tests == no bueno. could do regex, but want something like utils.TimeNow() that cc has
```

Comment:
The usual way I avoid this is to pass in the `now` timestamp into `generateHelpCommand`.